src/controllers/logout.controller.js

```diff
@@ -0,0 +1,24 @@
+import dotenv from 'dotenv';
+import DataSource from '../data-source.js';
+import handleResponse from './handleResponse.js';
+
+dotenv.config();
+
+const logout = async (req, res) => {
+	const { cookies } = req;
+	const accessToken = req.headers.authorization;
+	if (!cookies.jwt && !accessToken) {
+		return handleResponse(res, 200, { message: res.__('logout successful') });
+	}
+
+	/* c8 ignore next 7 */
+	if (cookies.jwt) {
+		await DataSource.query('TRUNCATE TABLE "RefreshToken"');
```

Comment:
There is no need to drop the while table while deleting one refresh token. 
You have to check for just a specifically a token and remove that.